onnxruntime/core/common/profiler.cc

```diff
@@ -33,14 +33,14 @@ void Profiler::StartProfiling(const std::string& file_name) {
 void Profiler::EndTimeAndRecordEvent(EventCategory category,
                                      const std::string& event_name,
                                      TimePoint& start_time,
-                                     const std::string& op_name,
+                                     const std::pair<std::string, std::string>& op_name,
```

Comment:
Thanks for this change. This pair can contain stuff besides op_name too, right? what's a good name for this parameter? tag?

Reply:
events?

Reply:
renamed to tags.